portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,17 +26,24 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> comments = new ArrayList<String>();    
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String jsonComments = convertToJsonUsingGson(comments);
     ArrayList<String> messages = new ArrayList<String>();
     messages.add("Good morning.");
     messages.add("Good afternoon.");
     messages.add("Good evening.");
 
     String jsonMessages = convertToJsonUsingGson(messages);
```

Comment:
It looks like this function doesn't modify the response at all, so I'm unsure what its purpose is. Correct me if I'm wrong, but I think that if you call this it won't do anything.